css/jquery.uls.lcd.css

```diff
@@ -43,6 +43,13 @@
 .grid .uls-language-block > ul {
 	margin: 0 0 20px 0;
 	padding: 0 16px;
+	/*
+	 * We don't want any visible bullets in this list. Not by default anyway.
```

Comment:
As the comment says, this is likely to break badges on Wikimedia because of the increased specificity.

Reply:
I reverted this change